multiregion/docs/multiregion.rst

```diff
@@ -7,17 +7,59 @@ Overview
 --------
 
 This tutorial describes the |mrrep| capability that is built directly into |cs|.
-The scenario for this tutorial is as follows:
 
-- Three regions: ``west``, ``central``, and ``east``
-- Broker naming convention: ``broker-[region]-[broker_id]``
+|mrrep| allow customers to run a single |ak-tm| cluster across multiple datacenters.
+Often referred to as a stretch cluster, |mrrep| replicate data between datacenters across regional availability zones.
+You can choose how to replicate data, synchronously or asynchronously, on a per |ak| topic basis.
+It provides good durability guarantees and makes disaster recovery (DR) much easier.
+
+Benefits:
+
+- Supports hybrid deployments of synchronous and asynchronous replication between datacenters
+- Consumers can leverage data locality for reading |ak| data, which means better performance and lower cost
+- Ordering of |ak| messages is preserved across datacenters
+- Consumer offsets are preserved
+- In event of a disaster in a datacenter, new leaders are automatically elected in the other datacenter, and applications proceed without interruption. This achieves RTO=0 and RPO=0
```

Comment:
This is accurate if sync replication is used. If async replication is used then the user needs to perform an unclean leader election if all of the sync replicas are down.

Reply:
@jsancio good point, I'm augmenting the text to clarify.